pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -280,6 +280,11 @@ func (co *ConvertOptions) RunConvert() (err error) {
 		edgeNodeNames = append(edgeNodeNames, node.GetName())
 	}
 
+	if len(co.CloudNodes) < 1 {
+		klog.Errorf("At least one cloud node should be provided!")
+		return
+	}
+
```

Comment:
Maybe check `co.CloudNodes` in func `Complete()` is better？

Reply:
Yes, you are right
My first attempt is to find out a master node if there is no one provided, yes keep it simple and explicit, and fail early